test/e2e/framework/helper/validation/validation.go

```diff
@@ -23,62 +23,21 @@ import (
 	"github.com/cert-manager/cert-manager/e2e-tests/framework/helper/validation/certificatesigningrequests"
 )
 
-func DefaultCertificateSet() []certificates.ValidationFunc {
```

Comment:
The Default... functions have been replaced by calling CertificateSetForUnsupportedFeatureSet with an empty set of unsupported features.